salt/apache/init.sls

```diff
@@ -57,3 +57,15 @@ apache2-utils:
 {{ apache(name, entry) }}
 {% endfor %}
 {% endif %}
+
+# Delay the Apache start up if the server has multiple IP addresses.
+{% if salt['pillar.get']('apache:wait_for_networking') == True %}
```

Comment:
Put the comment inside the `if`, so that it isn't added unless the if-statement runs.

```suggestion
{% if salt['pillar.get']('apache:wait_for_networking') %}
# Delay the Apache start up if the server has multiple IP addresses.
```

Reply:
Does this also check it is set to True? or is it just checking the `wait_for_networking` is there?
Functionally I suppose they work the same we're just being more explicit with the `== True`

Reply:
Yes, this works as expected in my testing.